.github/workflows/build-graphscope-dev-images.yml

```diff
@@ -75,12 +75,13 @@ jobs:
         # image tag
         tag=${{ github.event.inputs.v6d_version }}-${arch}
         # dev-wheel image
-        sudo docker tag graphscope/graphscope-dev:wheel-${tag} ${{ env.REGISTRY }}/graphscope/graphscope-dev:wheel-${tag}
-        sudo docker push ${{ env.REGISTRY }}/graphscope/graphscope-dev:wheel-${tag}
+        sudo docker tag graphscope/graphscope-dev:wheel-${tag} ${{ env.REGISTRY }}/graphscope/graphscope-dev:wheel-${{ github.event.inputs.v6d_version }}
+        sudo docker push ${{ env.REGISTRY }}/graphscope/graphscope-dev:wheel-${{ github.event.inputs.v6d_version }}
```

Comment:
You would better change the line `tag=${{ github.event.inputs.v6d_version }}` rather than modify the lines for docker tag and docker push.

Reply:
And leave a TODO here to explain that we are current lack of arm runners.

Reply:
Good advice！